scripts/calendar/asian_generator.py

```diff
@@ -415,9 +415,8 @@ def generate_data():
             holiday_data=holiday_data_str,
         )
 
-        f_name = Path("holidays/calendars") / OUT_FILE_NAME.format(cal_name=cal_name.lower())
-        with open(f_name, "w", encoding="UTF-8", newline="\n") as f:
-            f.write(class_str)
+        path = Path("holidays/calendars") / OUT_FILE_NAME.format(cal_name=cal_name.lower())
+        path.write_text(class_str)
```

Comment:
For `read_text()` and `write_text()`, it's necessary to specify UTF-8 encoding for proper work on Windows.

Reply:
The tests look good. Not sure if it's something env specific? Leaving this up to you as I didn't play as much w/ the calendar scripts.

Reply:
Because now we don't have non-ascii chars in README (and calendar generator's output). But we shouldn't rely on that :)

Reply:
Added `encoding="UTF-8"`.